### PR TITLE
Add release notes for v0.17

### PR DESCRIPTION
**Pull request by davidanthoff**
Lets put the release notes into the regular docs.

I would say all the contributors should just push their edits directly to this branch here, and then we review the final doc before we merge it into master.

### CODE REVIEW

File: docs/src/release-notes/v0_17.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The Variables pane in the integrated debugger now supports global variables:
 
 We added a run and debug button above the file editor area when you open a Julia file that makes it easier to run the currently active file:
 
-![run-debugger buttons](./0.17/run_debug_buttons.png)
+![run-debugger buttons](./0.17/run_debug_buttons_cropped.png)
```
Comment (aviatesk) [inline docs/src/release-notes/v0_17.md:17]: @davidanthoff if you don't like this cropping, please do reuse the original image :)

File: docs/src/release-notes/v0_17.md [hunk 2]
```diff
@@ -44,6 +44,14 @@ You can also look at the same information in a flamegraph by clicking the little
...
 Special thanks to [Valentin](https://github.com/vchuravy) and [Connor](https://github.com/connor4312) for their help with this feature!
 
+## Inline result styling improvements
+
+The stylings of inline result bubbles are greatly improved;
+the left-border color will make it easier to distinguish them from the actual code:
+
+- in dark theme ([Iceberg](https://marketplace.visualstudio.com/items?itemName=harg.iceberg)): ![inline bubbles dark](./0.17/inline_bubbles_dark.png)
+- in light theme ([Night Owl Light](https://marketplace.visualstudio.com/items?itemName=sdras.night-owl)): ![inline bubbles light](./0.17/inline_bubbles_light.png)
+
```
Comment (aviatesk) [inline docs/src/release-notes/v0_17.md:54]: If you think this section is verbose, I don't mind if you remove it

File: docs/src/release-notes/v0_17.md (hunk 1 above)
Comment (davidanthoff) [reply docs/src/release-notes/v0_17.md:17]: Perfect, way better than before!

File: docs/src/release-notes/v0_17.md (hunk 2 above)
Comment (davidanthoff) [reply docs/src/release-notes/v0_17.md:54]: I think the more we write the better :)

**Pull request merged**